CHANGES.rst

```diff
@@ -41,6 +43,9 @@ Misc
 ----
 
 
+4.5.2 (2022-05-04)
```

Comment:
not sure if this was an add from towncrier or manual, but it looks like this should be 4.5.0
```suggestion
4.5.0 (2022-05-04)
```